spec/reek/smells/smell_warning_spec.rb

```diff
@@ -86,6 +86,32 @@
     end
   end
 
+  context '#matches?' do
+    let(:uncommunicative) do
+      uncommunicative_name_detector = build(:smell_detector,
+                                            smell_type: 'UncommunicativeVariableName',
+                                            source: true)
+      build(:smell_warning, smell_detector: uncommunicative_name_detector,
+                            context: 'Dirty',
+                            message: "has the variable name '@s'",
+                            parameters: { test: 'something' })
+    end
```

Comment:
adding one to the end of the let ... end block - you want the it do ... end ones separated too?  the #yaml_hash example in the same file has them together ... assuming that's the pattern I should follow